bindings/flow/tester/Tester.actor.h

```diff
@@ -35,10 +35,17 @@
 #include "flow/actorcompiler.h"  // This must be the last #include.
 
 #define LOG_ALL 0
-#define LOG_INSTRUCTIONS LOG_ALL || 0
-#define LOG_OPS LOG_ALL || 0
-#define LOG_DIRS LOG_ALL || 0
-#define LOG_ERRORS LOG_ALL || 0
+#if LOG_ALL == 1
```

Comment:
I'm curious what the motivation was for the weird syntax, but git blame didn't get me anywhere. New version seems ok to me

Reply:
The intent of the old code was that you could turn on logging for all categories or any individual category by changing a 0 to a 1 for the relevant category.

Is the below supposed to have `LOG_ALL` in the define (e.g. `#define LOG_OPS LOG_ALL 1`)

Reply:
This was a weird "me using emacs macro incorrectly for refactoring"-bug...

IMHO: these shouldn't be macros in the first place. So I just change them to `constexpr` variables with shuts up compiler warnings. This is imho also much nicer